starknet-providers/src/jsonrpc/mod.rs

```diff
@@ -111,8 +111,10 @@ struct FeltArray(#[serde_as(as = "Vec<UfeHex>")] pub Vec<FieldElement>);
 struct EventFilterWithPage {
     #[serde(flatten)]
     filter: EventFilter,
-    page_size: u64,
-    page_number: u64,
+    /// a pointer to the last element of the delivered page, use this token in a subsequent query to obtain the next page
```

Comment:
No comment/docs lines should exceed 100 chars for readability. Hard-wrap it into the next line. Please also fix other occurrences.